docs/specs/servers/blockchain/blockchain-server-api.md

```diff
@@ -276,3 +276,62 @@ The POST request body should be in JSON format with the following structure:
 
 * `400 Bad Request` - some parameters in request body were missed or wrong.
 * `401 Unauthorized` - projectID verification error.
+
+## SWAP
+
+Endpoints to make SWAP transactions
+
+### List of tokens available for SWAP
+
+`GET /v1/swap/tokens`
+
+#### Path parameters
+
+* `projectId` - is the project identifier
```

Comment:
this is required, correct? do we differentiate between required / not required params?

Reply:
They all seem to be required.